crates/sui-bridge-indexer/src/main.rs

```diff
@@ -166,14 +173,6 @@ async fn start_processing_eth_events(config: &sui_bridge_indexer::config::Config
 
     // capture unfinalized blocks
 
-    let eth_client = Arc::new(
-        EthClient::<ethers::providers::Http>::new(
-            &config.eth_rpc_url,
-            HashSet::from_iter(vec![bridge_address]),
-        )
-        .await?,
-    );
-
```

Comment:
cc @Bridgerz  we shouldn't need another instance for EthClient

Reply:
Makes sense... my b

Reply:
Hi